### PR TITLE
Adding fixes for signedness warnings in test/tools/yulInterpreter

### PR DESCRIPTION
**Pull request by mijovic**

### CODE REVIEW

**Review by leonardoalt — DISMISSED** [review]

File: test/tools/yulInterpreter/EwasmBuiltinInterpreter.cpp [hunk 1]
```diff
@@ -87,9 +87,9 @@ u256 EwasmBuiltinInterpreter::evalBuiltin(YulString _fun, vector<u256> const& _a
 			copyZeroExtended(
 				m_state.memory,
 				m_state.code,
-				size_t(_arguments.at(0)),
-				size_t(_arguments.at(1) & size_t(-1)),
-				size_t(_arguments.at(2))
+				static_cast<size_t>(_arguments.at(0)),
+				static_cast<size_t>(_arguments.at(1)) & numeric_limits<size_t>::max(),
```
Comment (ekpyron) [inline test/tools/yulInterpreter/EwasmBuiltinInterpreter.cpp]:
```suggestion
				static_cast<size_t>(_arguments.at(1) & numeric_limits<size_t>::max()),
```
Right? That's what we actually want, isn't it?

**Review by ekpyron — CHANGES_REQUESTED** [review]

**Review by chriseth — APPROVED** [review]

**Review by ekpyron — APPROVED** [review]

File: test/tools/yulInterpreter/EwasmBuiltinInterpreter.cpp [hunk 2]
```diff
@@ -87,9 +87,9 @@ u256 EwasmBuiltinInterpreter::evalBuiltin(YulString _fun, vector<u256> const& _a
 			copyZeroExtended(
 				m_state.memory,
 				m_state.code,
-				size_t(_arguments.at(0)),
-				size_t(_arguments.at(1) & size_t(-1)),
-				size_t(_arguments.at(2))
+				static_cast<size_t>(_arguments.at(0)),
```
Comment (axic) [inline test/tools/yulInterpreter/EwasmBuiltinInterpreter.cpp:90]: Why was this file changed to `static_cast`, but not the file above?
